packages/yourloops/test/integration/pages/trends/trends-view-common.spec.tsx

```diff
@@ -91,6 +97,12 @@ describe('Trends view for anyone', () => {
       await checkTrendsTimeInRangeStatsWidgets()
       await checkTimeInRangeStatsTitle()
     })
+
+    it('with the Cbg data', async () => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches to further communicate your intent.  -->
**🎯 suggestion (blocking):** ‏`it` test must begin by the word 'should'. I think you can move this one into the describe 'with all kind of data'